recipe/meta.yaml

```diff
@@ -24,17 +24,16 @@ requirements:
   run:
     - python >=3.5
     - joblib
-    - scikit-learn <=0.22.1
-    - networkx
-    - xgboost
+    - numpy
     - pandas
-    - zlib
-    - requests
-    - rdkit
+    - pillow # pillow will remove from v2.4.0
```

Comment:
Pillow was hard dependecies in deepchem 2.3.0, so I added it.
I think this is work around and we will remove pillow in the next 2.4.0